### PR TITLE
Feature/mini map

### PR DESCRIPTION
**Pull request by oconn**
# PR for Issue #210.

## PR Status

**READY**

## Changes proposed in the pull request:
- Add fog 
- Add toggle for wombat / full view

## Breaking changes?
- None

## Screenshot of feature:

image

@emilyseibert @oconn @dehli

### CODE REVIEW

File: src/cljs/wombats_web_client/components/arena.cljs [hunk 1]
```diff
@@ -74,6 +74,7 @@
     :shot      1
     :explosion 2
     :smoke     3
+    :fog       3
```
Comment (dehli) [inline src/cljs/wombats_web_client/components/arena.cljs]: I would make this a 4 if we want fog always drawn on top of smoke (unless we want smoke on top of fog)

**Comment by oconn** [comment]
@dehli Got rid of the fog `meta-value`. It's actually not being used because fog is not in metadata.

File: src/cljs/wombats_web_client/panels/simulator.cljs [hunk 2]
```diff
@@ -57,11 +57,21 @@
...
 (defn- render
-  [sim-pane templates wombats active-frame stack-trace simulator-state]
+  [sim-pane
+   templates
+   wombats
+   active-frame
+   stack-trace
+   simulator-display-mini-map
+   simulator-mini-map
+   simulator-state]
```
Comment (emilyseibert) [inline src/cljs/wombats_web_client/panels/simulator.cljs]: should these be keys?

Comment (oconn) [inline src/cljs/wombats_web_client/panels/simulator.cljs]: These are all realized values from the db. Not 100% sure what the question is?

File: src/cljs/wombats_web_client/panels/simulator.cljs [hunk 3]
```diff
@@ -82,4 +94,6 @@
                                @wombats
```
Comment (emilyseibert) [inline src/cljs/wombats_web_client/panels/simulator.cljs]: should we pass all these params in as keys? normally when there are a lot of params we do

**Comment by oconn** [comment]
@emilyseibert I think I see what you're saying. Is that better?

**Comment by emilyseibert** [comment]
yes sorry if i wasn't very clear but much better

**Pull request merged**